src/CallHandler.tsx

```diff
@@ -138,22 +139,11 @@ export enum PlaceCallType {
     ScreenSharing = 'screensharing',
 }
 
-function getRemoteAudioElement(): HTMLAudioElement {
-    // this needs to be somewhere at the top of the DOM which
-    // always exists to avoid audio interruptions.
-    // Might as well just use DOM.
-    const remoteAudioElement = document.getElementById("remoteAudio") as HTMLAudioElement;
-    if (!remoteAudioElement) {
-        console.error(
-            "Failed to find remoteAudio element - cannot play audio!" +
-            "You need to add an <audio/> to the DOM.",
-        );
-        return null;
-    }
-    return remoteAudioElement;
+export enum CallHandlerEvent {
+    CallsChanged = "calls_changed",
 }
 
-export default class CallHandler {
+export default class CallHandler extends EventEmitter {
```

Comment:
Hmm, OK - I guess this is basically turning CallHandler into a store, which is probably correct in our current definition of stores and what it should be. If you're doing this would you mind also replacing the CallChangeRoom dispatch so we don't have a mix of both?